### PR TITLE
L1-315: Updated payout_stakers bench with latest main

### PR DESCRIPTION
**Pull request by Marcin-Radecki**
# Description

This PR updates `payout_stakers` create so it runs with latest `main`, and how to run it against `r-13.4` release. Ultimately, this test should end up in nightly CI. 

## Type of change

Please delete options that are not relevant.

- New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: benches/payout-stakers/README.md [hunk 1]
```diff
@@ -2,8 +2,17 @@
 
 This is a performance, e2e test for checking rewards for upper bound limit of nominators.
 
-Run by 
+## How to run
+
+1. Start local chain, e.g. via `./scripts/run_nodes.sh -v 1 -n 1 --dont-build`, where `-n`  denotes number of RPC nodes to start
+2. Set `aleph-client` in `Cargo.toml` to point to commit sha which your local `aleph-node` was built from. E.g. below builds `aleph-client` based on `r-13.4` version:
+```
```
Comment (kostekIV) [inline benches/payout-stakers/README.md:9]: :(

**Review by kostekIV — APPROVED** [review]

File: benches/payout-stakers/src/main.rs [hunk 2]
```diff
@@ -236,33 +275,35 @@ async fn nominate_validator(
...
-        let validator_address = address.to_string();
-        handles.push(tokio::spawn(async move {
-            let connection = SignedConnection::new(&validator_address, validator).await;
-            connection
-                .bond(MIN_VALIDATOR_BOND, TxStatus::InBlock)
-                .await
-                .unwrap();
-        }));
+/// We assume stash == validator == controller.
+async fn bond_validators_funds_and_choose_controllers(address_ip_without_port: &str, validators: Vec<KeyPair>) {
```
Comment (timorleph) [inline benches/payout-stakers/src/main.rs:290]: Is `and_choose_controllers` still in any way accurate here?

File: benches/payout-stakers/src/main.rs [hunk 3]
```diff
@@ -21,7 +25,7 @@ use rand::{thread_rng, Rng};
 use sp_keyring::AccountKeyring;
 
 // testcase parameters
-const NOMINATOR_COUNT: u32 = MAX_NOMINATORS_REWARDED_PER_VALIDATOR;
+const NOMINATOR_COUNT: u32 = 2 * MAX_NOMINATORS_REWARDED_PER_VALIDATOR;
```
Comment (timorleph) [inline benches/payout-stakers/src/main.rs:28]: Perhaps a comment as to why the `2 *` would be nice. Are we testing that some nominators don't get any rewards?

File: bin/chain-bootstrapper/src/chain_spec/builder.rs [hunk 4]
```diff
@@ -116,7 +116,7 @@ fn generate_genesis_config(
         "staking": {
             "forceEra": Forcing::NotForcing,
             "validatorCount":  account_session_keys.len() as u32,
-            "minimumValidatorCount": 4,
+            "minimumValidatorCount": 1,
```
Comment (timorleph) [inline bin/chain-bootstrapper/src/chain_spec/builder.rs:119]: I hate this and I don't trust anything to work with <4 validators, but you can just resolve this comment as I think I have lost this fight already. :P

**Review by timorleph — APPROVED** [review]
Some minor comments, other than that looks good.

File: benches/payout-stakers/src/main.rs (hunk 2 above)
Comment (Marcin-Radecki) [reply benches/payout-stakers/src/main.rs]: Good point, not really, I'll fix naming

File: benches/payout-stakers/src/main.rs (hunk 3 above)
Comment (Marcin-Radecki) [reply benches/payout-stakers/src/main.rs:33]: I added a comment. This particular bench fails when run on r-13.4 version, but it is OK, as those changes prepare this crate to be run in the CI, which is based on `main`, that has paged exposure logic already. In the paged exposure world, all nominators are paid, regardless of how many of them are. In subsequent PR we should add a nighty workflow that runs this bench against `aleph-node` from `main`, and modify this file to call `payout_stalers` in a loop, until `Error::AlreadyClaimed` is received.

**Pull request merged**